src/Components/Facility/Consultations/Feed.tsx

```diff
@@ -91,59 +83,54 @@ export const Feed: React.FC<IFeedProps> = ({ consultationId, facilityId }) => {
       setCamTimeout(0);
     }, 5000);
     return () => clearTimeout(timeout);
-  }, [cameraState]);
+  }, [cameraState, cameraConfig]);
 
   const isIOS = /iPad|iPhone|iPod/.test(navigator.userAgent);
 
   const liveFeedPlayerRef = useRef<HTMLVideoElement | ReactPlayer | null>(null);
 
-  const fetchData = useCallback(
-    async (status: statusType) => {
-      setIsLoading(true);
-      const res = await dispatch(getConsultation(consultationId));
-      if (!status.aborted && res?.data) {
-        const consultation = res.data as ConsultationModel;
+  const { loading: getConsultationLoading } = useQuery(routes.getConsultation, {
+    pathParams: { id: consultationId },
+    onResponse: ({ res, data }) => {
+      if (res && res.status === 200 && data) {
+        const consultation = data as ConsultationModel;
```

Comment:
Why typecase when you already have defined the proper type in the route definition itself?

```suggestion
```